pkg/harvester/edit/harvesterhci.io.virtualmachinetemplateversion.vue

```diff
@@ -242,7 +242,14 @@ export default {
         <CpuMemory :cpu="cpu" :memory="memory" :disabled="isConfig" @updateCpuMemory="updateCpuMemory" />
 
         <div class="mb-20">
-          <SSHKey v-model="sshKey" :namespace="templateValue.metadata.namespace" :disable-create="isView" :mode="mode" @update:sshKey="updateSSHKey" />
+          <SSHKey
+            v-model="sshKey"
+            :create-namespace="true"
+            :namespace="templateValue.metadata.namespace"
+            :disable-create="isView"
```

Comment:
`disable-create` prop is legacy code, but it's used as boolean.

Reply:
Let's fix it in another refactoring PR. I see that other components us `disableCreate` and I'm not sure from what it comes in some cases.

Reply:
sure, let's keep this focus on fixing https://github.com/harvester/harvester/issues/5419.